base/interactiveutil.jl

```diff
@@ -152,7 +152,9 @@ end
         systemerror(:CloseClipboard, 0==ccall((:CloseClipboard, "user32"), stdcall, Cint, ()))
         plock = ccall((:GlobalLock, "kernel32"), stdcall, Ptr{UInt16}, (Ptr{UInt16},), pdata)
         systemerror(:GlobalLock, plock==C_NULL)
-        s = utf8(utf16(plock))
+        len = 0
+        while unsafe_load(plock, len+1) != 0; len += 1; end
```

Comment:
Maybe `len = ccall(:wcslen, Csize_t, (Ptr{Cwchar},), plock)` would be clearer?   Better yet, add a method:
```jl
@windows_only utf16to8(p::Ptr{UInt16}, len=ccall(:wcslen, Csize_t, (Ptr{UInt16},), p)) = utf16to8(pointer_to_array(p, len))
```
method?
```